### DISCUSSION TITLE
Dependency not picked up on updates

### DISCUSSION THREAD

**Discussion by raamcosta** [renovatebot/renovate#21150] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

To have renovatebot consider compose destinations library:

https://github.com/raamcosta/compose-destinations

### What you tried so far.

This issue was reported to me (the author of the said library) so I haven't actually tried it yet.

I'm just curious if there is any obvious thing I may be doing wrong...
I guess one unique thing is that my library has three active versions and each of them is suffixed with "-beta".

For example, a typical update looks like this:

1.5.30-beta -> 1.5.31-beta
1.6.30-beta -> 1.6.31-beta
1.7.30-beta -> 1.7.31-beta

Notice that I consider "beta" a state of the whole library and not a version-dependent state as I know some others do. I may be doing it wrong, but I want to convey the message that the whole dependency is in a beta state, not a particular version of it.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Users can set a package rule with matchPackageNames=X, and ignoreUnstable=false. In other words: stop ignoring unstable updates for package X
